lib/pbench/agent/tool_meister.py

```diff
@@ -1947,7 +1925,7 @@ def main(argv):
             break
     else:
         _sep = "" if not _path else ":"
-        os.environ["PATH"] = f"{_path}{_sep}{_prog.parent}"
+        os.environ["PATH"] = f"{_path}{_sep}{prog.parent}"
```

Comment:
Consider using [`os.pathsep`](https://docs.python.org/3/library/os.html#os.pathsep) instead of `":"`.